.eslintrc.json

```diff
@@ -33,7 +33,7 @@
         "leadingUnderscore": "forbid",
         "trailingUnderscore": "forbid",
         "filter": {
-          "regex": "^(__html|_jsx)$",
+          "regex": "^(__html|_jsx|_jsxs|_Fragment)$",
```

Comment:
At this point I'd just remove this whole thing, leave only `__html` and add this:
```ts
/* eslint-disable @typescript-eslint/naming-convention */
```
to `runtime.ts`